qiskit_experiments/framework/base_experiment.py

```diff
@@ -319,7 +328,7 @@ def _default_experiment_options(cls) -> Options:
         # that experiment and their default values. Only options listed
         # here can be modified later by the different methods for
         # setting options.
-        return Options()
```

Comment:
The docstring should be updated to include the option so it will be rendered in the API docs (actually not sure whether that will work with the current doc settings since `separate_jobs` isn't showing up as an option for `BatchExperiment`).